executor/agents/agno/mcp_manager.py

```
@@ -199,6 +231,12 @@ async def cleanup_tools(self) -> None:
                     await tools.disconnect()
             except Exception as e:
                 logger.warning(f"Failed to disconnect MCP tools: {str(e)}")
```

Ruff (0.14.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)